recipes/bioconductor-ballgown/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: bioconductor-ballgown
+  version: 2.4.2
```

Comment:
I'm not sure what you mean with your version question?
Are you referring to this version? It should be 2.2.0? No?